scripts/binaryninja/manticore_viz/__init__.py

```diff
@@ -0,0 +1,201 @@
+import fcntl
+import os
+import time
+import threading
+
+from binaryninja import PluginCommand, HighlightStandardColor, log
+from binaryninja.interaction import (
+    get_open_filename_input, get_directory_name_input, get_choice_input
+)
+import binaryninja.enums as enums
+
+blue = HighlightStandardColor.BlueHighlightColor
+black = HighlightStandardColor.BlackHighlightColor
+white = HighlightStandardColor.WhiteHighlightColor
+clear = HighlightStandardColor.NoHighlightColor
+# renew interval
+interval = 1.5
+
+class Singleton(type):
+    _instances = {}
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
+
+class TraceVisualizer(object):
```

Comment:
The idea was that you only want a single class to be called whenever you instantiate things. E.g. you want to recall what you highlighted (also perhaps user's selection of a workspace?) so that you can run `clear` on it